src/bin/CMakeLists.txt

```diff
@@ -44,12 +49,14 @@ configure_file(msquic-config.cmake.in ${CMAKE_BINARY_DIR}/msquic-config.cmake)
 
 install(FILES ${CMAKE_BINARY_DIR}/msquic-config.cmake DESTINATION ${msquic_dest})
 
-install(EXPORT msquic DESTINATION ${msquic_dest})
+if(BUILD_SHARED_LIBS)
+    install(EXPORT msquic DESTINATION ${msquic_dest})
+endif()
 
 if(WIN32)
     add_library(msquic.lttng INTERFACE)
 elseif(QUIC_ENABLE_LOGGING)
-    add_library(msquic.lttng SHARED $<TARGET_OBJECTS:platform.clog.provider> $<TARGET_OBJECTS:core.clog.provider>)
+    add_library(msquic.lttng $<TARGET_OBJECTS:platform.clog.provider> $<TARGET_OBJECTS:core.clog.provider>)
```

Comment:
This actually always needs to be a shared library, because of how this works. But this doesn't actually affect user code in the normal case.